collection-scripts/gather_network_logs

```diff
@@ -4,15 +4,14 @@ NETWORK_LOG_PATH=${OUT:-"${BASE_COLLECTION_PATH}/network_logs"}
 
 mkdir -p "${NETWORK_LOG_PATH}"/
 
-function gather_multus_data { 
-  #Should we store this on a per namespace basis?
-  oc get net-attach-def -o yaml --all-namespaces > "${NETWORK_LOG_PATH}"/net_attach_def 2>&1 & PIDS+=($!)
+function gather_multus_data {
+  local resources=(ippools.whereabouts.cni.cncf.io overlappingrangeipreservations.whereabouts.cni.cncf.io \
+      net-attach-def multi-networkpolicy)
 
-  oc describe ippools.whereabouts.cni.cncf.io -A > "${NETWORK_LOG_PATH}"/ippools 2>&1 & PIDS+=($!)
-
-  oc describe overlappingrangeipreservations.whereabouts.cni.cncf.io -A > "${NETWORK_LOG_PATH}"/overlappingrangeipreservations  2>&1 & PIDS+=($!)
-
-  oc get multi-networkpolicy -o yaml --all-namespaces > "${NETWORK_LOG_PATH}"/multi_networkpolicy 2>&1 & PIDS+=($!)
+  for resource in "${resources[@]}"; do
+    oc adm inspect --dest-dir "${BASE_COLLECTION_PATH}" "${resource}" --all-namespaces 2>&1 || true & PIDS+=($!)
+    oc describe "${resource}" -A > "${NETWORK_LOG_PATH}"/"${resource}" || true 2>&1 & PIDS+=($!)
```

Comment:
@andreaskaris is there a more machine readable command we could use over `describe`?

Reply:
I'm only collecting the describe because it was part of the old set of data that was collected; the oc adm inspect should be the machine readable part. With that said, @npinaeva also rightfully pointed out https://bugzilla.redhat.com/show_bug.cgi?id=2023295 and that we shouldn't gather cluster wide resources in the first place